lms/djangoapps/learner_dashboard/serializers.py

```diff
@@ -0,0 +1,35 @@
+"""
+Serializers for the Learner Dashboard
+"""
+
+from rest_framework import serializers
+
+
+class EdxSerializer(serializers.Serializer):
```

Comment:
Should this be some other name? PlatformSettingsSerializer? EdxSerializer is a pretty generic name